.circleci/config.yml

```diff
@@ -272,8 +274,9 @@ jobs:
             mkdir -p ~/.docker/trust/private
             cp .circleci/content-trust.key ~/.docker/trust/private/b056f84873aa0be205dfe826afa6e7458120c9569dd19a2a84154498fb1165d5.key
 
-            linuxkit pkg push --nobuild pkg/cri-containerd
             linuxkit pkg push --nobuild pkg/kubelet
+            linuxkit pkg push --nobuild pkg/cri-containerd
+            linuxkit pkg push --nobuild pkg/kube-e2e-test
```

Comment:
Why did you move cri-containerd below kubelet? They were alphabetic before. Suppose it isn't a big deal.

Reply:
They were in different order in some places, so I was trying to do list them in the same order everywhere..

Reply:
I guess it was the other places which were buggy then ;-)

Lets not worry about it, if someone enthusiastic soul decides they want to sort them later (and add comments to keep them sorted) then they can.